bundles/org.openhab.binding.publictransportswitzerland/src/main/resources/ESH-INF/i18n/publictransportswitzerland_en_US.properties

```diff
@@ -0,0 +1,15 @@
+# binding
+binding.publictransportswitzerland.name = Public Transport Switzerland
+binding.publictransportswitzerland.description = Connects to the "Swiss public transport API" to provide real-time public transport information.
+
+# thing types
+thing-type.publictransportswitzerland.stationboard.label = Stationboard
+thing-type.publictransportswitzerland.stationboard.description = Upcoming departures for a single station.
+
+# thing type config description
+thing-type.config.publictransportswitzerland.stationboard.station.label = Station
+thing-type.config.publictransportswitzerland.stationboard.station.description = The name of the station
+
+# channel types
+channel-type.publictransportswitzerland.csv.label = CSV
+channel-type.publictransportswitzerland.csv.description = A CSV representation
```

Comment:
Such of thing is not required for English.
It is required only if you provide another language than English.

Reply:
So even if I add a second file for German, I don't need to have one for English?

Reply:
Yes.
The values for English language are directly coming from your XML defining the binding and the thing type.

Reply:
6db3a3b